src/pages/[platform]/build-a-backend/auth/connect-your-frontend/sign-up/index.mdx

```diff
@@ -285,15 +285,26 @@ func signUp(username: String, password: String, email: String, phonenumber: Stri
 
 The `signUp` API response will include a `nextStep` property, which can be used to determine if further action is required. It may return the following next steps:
 
-<InlineFilter filters={["angular", "javascript", "react", "nextjs", "vue", "android"]}>
-- `CONFIRM_SIGN_UP` - The sign up needs to be confirmed by collecting a code from the user and calling `confirmSignUp`.
-- `DONE` - The sign up process has been fully completed.
-- `COMPLETE_AUTO_SIGN_IN` - The sign up process needs to complete by invoking the `autoSignIn` API.
+<InlineFilter filters={["angular", "javascript", "react", "nextjs", "vue"]}>
```

Comment:
missing react-native in this filter set

Reply:
Added, [found a couple more spots.](https://github.com/aws-amplify/docs/pull/8023/commits/c02906487ddf48e8b1c515629dfbfe02c81caacd)